CONTRIBUTING.md

```diff
@@ -10,3 +31,14 @@ in all of your interactions with the project.
 3. Write a clear commit message
 4. Open a [Pull Request](https://github.com/panther-labs/panther-analysis/pulls)
 5. Incorporate feedback and merge once you have the sign-off of other code owners. If you do not have permission, you may request a reviewer to merge it for you.
```

Comment:
also don't think this changed here, but on #5, sugg change to:
5. Once your PR has been approved by code owners, if you have merge permissions, merge it. If you do not have merge permissions, leave a comment requesting a code owner merge it for you.